saskatoon/harvest/models.py

```diff
@@ -774,6 +774,11 @@ def __str__(self):
     sender=Property
 )
 
+models.signals.pre_save.connect(
```

Comment:
Just a little question, why using `pre_save` and not `post_save` here? 
If there is an error while calling `save()`, I suppose the email would have been triggered anyway.

Tell me what you think,

Reply:
I think `pre_save` is used to detect the change in status (`original_instance.pending`). After saving the object you'd loose track of what the former value of `pending` used to be. An alternative to avoid relying on signals would have been to manually store `self.pending` in the model's `__init__` and compare it to the new value in the `save()` method but i find the `pre_save` pretty elegant.

Reply:
ok very well, this was a misunderstanding.